test/rest-smoketest.sh

```diff
@@ -200,13 +217,28 @@ dotest "204::*::*" "${obj1_vers1}" -X DELETE
 dotest "404::*::*" "${obj1_vers1}"
 dotest "409::*::*" /ns-${RUNKEY}/foo2/obj1
 
-# test chunk upload
-cat > ${TEST_DATA} <<EOF
+# test chunk upload (S3 requires at least 5MB chunks)
+if [[ $DEPLOYMENT == "amazons3" ]]
+then
+    # generate 5MB + file
+    dd if=/dev/urandom bs=6291456 count=1 of=/tmp/dummy-${RUNKEY}
```

Comment:
Given that the dotest function will print out response content to the console, I think you should not use random data directly as the test data.  One option would be to base64 encode the random test data that you write into TEST_DATA.